app/src/state/editor/edit.ts

```diff
@@ -325,26 +325,8 @@ export const copySelectionText = createAsyncActionWithReducer<EditorState>(
       });
```

Comment:
the code above seems redundant to me?

Reply:
Redundant to what?

Reply:
Ahh, right. We added this code in `selectedItems`